dotnet/src/Planners/Planners.Handlebars/Handlebars/Exceptions/HandlebarsPlanCreationException.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Planning.Handlebars;
+
+/// <summary>
+/// Provides extension methods for rendering Handlebars templates in the context of a Semantic Kernel.
+/// </summary>
+internal sealed class HandlebarsPlanCreationException : KernelException
+{
+    /// <summary>
+    /// Gets the prompt template used to generate the plan.
+    /// </summary>
+    public string? CreatePlanPrompt { get; set; } = null;
+
+    /// <summary>
+    /// Completion results from the model, generally, the proposed handlebars template representing the plan.
+    /// </summary>
+    public ChatMessageContent? ModelResults { get; set; } = null;
+
+    public HandlebarsPlanCreationException() : base()
+    {
+    }
+
+    public HandlebarsPlanCreationException(string? message) : base(message)
+    {
+    }
+
+    public HandlebarsPlanCreationException(string? message, System.Exception? innerException) : base(message, innerException)
+    {
+    }
```

Comment:
Why do we need these extra constructors?

Reply:
To comply with linter rules since we're extending the `KernelException` class

![image](https://github.com/microsoft/semantic-kernel/assets/125500434/9f4df598-dcbd-468d-a97b-12c6db3ae583)